apps/microtvm/ethosu/README.md

```diff
@@ -72,15 +73,15 @@ the locations for these can be specified as arguments to run_demo.sh, for exampl
 ```
 
 This will:
-- Download a quantized mobilenet v1 model
-- Use tvmc to compile the model for Cortex(R)-M55 CPU and Ethos(TM)-U55 NPU
-- Download an image of a kitten to run the model on
+- Download a quantized (int8) mobilenet v2 model
+- Use tvmc to compile the model for Cortex(R)-M55 CPU, Ethos(TM)-U55 NPU and CMSIS-NN
+- Download an image of a penguig to run the model on
```

Comment:
I don't think this hybrid is the one we're testing for.

Reply:
Thanks @Mousius , I've fixed that now :)